Extensions/find_inactives.js

```diff
@@ -290,7 +307,7 @@ XKit.extensions.find_inactives = new Object({
 
 		}
 
-		$(".find-inactives-blog").click(function() {
+		$(".find-inactives-blog-link").click(function() {
```

Comment:
Tested it, it doesn't seem to break. The click should open the blog, right? That's what it does